src/_layouts/base.html

```diff
@@ -12,7 +12,7 @@
     <meta name="theme-color" content="#ffffff" />
     {% unless page.strip_fonts == true %}
      <link rel="preconnect" href="https://fonts.googleapis.com" />
-     <link rel="preconnect" href="https://use.fontawesome.com" />
+     <link rel="preload" href="https://use.fontawesome.com/releases/v5.15.1/js/all.js" as="script" />
```

Comment:
PTAL @parlough @khanhnwin 

I'm curious how this preload will be keep in sync with the version of fontawesome that is included elsewhere in this site?

Reply:
Perhaps just an inline comment saying the URL needs to be updated here and below if upgrading Font Awesome.

As for the other version of font awesome we include a copy of, we mostly don't need to worry about updating it and I'd like to remove that usage soon.